loqusdbapi/exceptions.py

```diff
@@ -0,0 +1,12 @@
+class LoqusdbAPIError(Exception):
+    def __init__(self, message: str = ""):
+        super(LoqusdbAPIError, self).__init__()
+        self.message = message
+
+
+class VCFParserError(LoqusdbAPIError):
+    """Raise when provided VCF is malformed"""
```

Comment:
no need to set pass here?!

Reply:
Nope, pass is implied. But setting doesnt hurt!